cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -25,6 +25,97 @@ function get_md5sum() {
     echo "${sig:?}"
 }
 
+#
+# Get pip
+# - @param1 python_exe - the python executable to install PIP for
+function get_pip() {
+    local python_exe=${1:?}
+
+    echo -e "--- Python: ${python_exe:?}"
+
+    # fetch get-pip.py
+    echo -e "--- curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py"
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+
+    get_pip_args=(
+        --user
+        --proxy="http://wwwproxy.sandia.gov:80"
+        --no-setuptools
+        --no-wheel
+    )
+    echo -e ""
+    echo -e "--- ${python_exe:?} ./get-pip.py ${get_pip_args[@]}"
+    ${python_exe:?} ./get-pip.py ${get_pip_args[@]}
+}
+
+#
+# Install Python pacakges using pip
+#
+# - @param1 pip_exe - the pip binary to use, i.e., pip3.
+#
+function get_python_packages() {
+    local pip_exe=${1:?}
+
+    echo -e "--- Pip   : ${pip_exe:?}"
+
+    pip_args=(
+        --use-feature=2020-resolver
+        configparser
+    )
+    echo -e "--- ${pip_exe:?} install --user ${pip_args[@]}"
+    ${pip_exe:?} install --user ${pip_args[@]}
+}
+
+
+
+
+# Load the right version of Git / Python based on a regex
+# match to the Jenkins job name.
+function bootstrap_modules() {
+    echo -e "PRDriver> ---------------------------------------"
+    echo -e "PRDriver> Bootstrap environment modules Start"
+    echo -e "PRDriver> ---------------------------------------"
+
+    cuda_regex=".*(_cuda_).*"
+    ride_regex=".*(ride).*"
```

Comment:
Do you want to change the ride job names to include some kind of delimiters around `ride` for increased robustness?

Reply:
This particular code is generally cut and paste in from the old scripts... I pretty much left this stuff as is.  I'm thinking we can do a cleanup pass on PullRequestLinuxDriver.sh after we remove the Python2 dependency and get the updated PR script in.

But I do agree that we do need to do some pass at some point to handle system-specific issues more cleanly. Currently stuff like this is spread around in a few places.